bridge/qb/server/player.lua

```diff
@@ -88,7 +88,8 @@ end
 ---@param Offline boolean
 ---@return Player player
 function playerObj.CreatePlayer(playerData, Offline)
-    AddDeprecatedFunctions(exports.qbx_core:CreatePlayer(playerData, Offline))
+    ---@diagnostic disable-next-line: return-type-mismatch
```

Comment:
The return type shouldn't mismatch. I think the return type Just needs to be changed to Player?